recipes/czmq/all/conanfile.py

```diff
@@ -42,15 +42,15 @@ def configure(self):
             del self.options.fPIC
 
     def requirements(self):
-        self.requires("openssl/1.1.1d")  # zdigest depends on openssl
-        self.requires.add("zeromq/4.3.2")
+        self.requires("openssl/1.1.1f")  # zdigest depends on openssl
```

Comment:
```suggestion
        self.requires("openssl/1.1.1g")  # zdigest depends on openssl
```

Since you already re-triggered CI for something else you could update this once more :)

Reply:
This is non-stop! 😄